pegasus/emails/hoc_signup_2022_receipt_en.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Are these receipt emails exact copies from last year? Do we generally like just having a clear record of what email was sent each year (even if it duplicates some stuff)?

Reply:
And then gets duplicated again in test fixtures...feels not great, but probably out of scope for this work.

Reply:
Both good points.

I also pondered just sending the older emails but figured we might want to customize these emails at some point during this hoc season.

Some interesting context on the fixtures [here](https://github.com/code-dot-org/code-dot-org/pull/33362).

Reply:
I seem to recall last time I did these that we used to put the dates into the emails, but it looks like we don't do that any more? I think it could be a nice cleanup to just have one email if we don't need to be sending the old email and writing the new email at the same time (but agree out of scope of this pr).